skylib/k8s.bzl

```diff
@@ -265,6 +266,32 @@ def k8s_deploy(
             image_tag_patches = image_tag_patches,
             tags = tags,
         )
+        kustomize(
+            name = name + "_gitops",
```

Comment:
created a duplicate of the above kustomize function call, but create another target that will be used exclusively for gitops